package.json

```diff
@@ -7,6 +7,9 @@
 		"type": "git",
 		"url": "https://github.com/DoctorMcKay/node-steam-tradeoffer-manager.git"
 	},
+	"scripts": {
+		"lint": "./node_modules/.bin/eslint ."
```

Comment:
@scholtzm I've added it because I didn't have `./node_modules/.bin/` in my PATH variable, that's why I've added it.
About second one: I actually didn't know about that one before, yeah, that's good idea.